src/js/binaryen.js-post.js

```diff
@@ -121,6 +123,7 @@ function initializeConstants() {
     'TailCall',
     'ReferenceTypes',
     'Multivalue',
+    'GC',
```

Comment:
Tried to get around adding the feature to keep the diff small, but found that quite a few moving parts depend on it, so I ended up adding it here.